reporter/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -536,6 +536,26 @@ <h3>Packages</h3>
                   <p>Unknown time [ERROR]: FileCounter - DownloadException: No source artifact URL provided for 'Gradle:org.ossreviewtoolkit.gradle.example:lib:1.0.0'.<br>Caused by: DownloadException: No VCS URL provided for 'Gradle:org.ossreviewtoolkit.gradle.example:lib:1.0.0'. Please make sure the release POM file includes the SCM connection, see: https://docs.gradle.org/current/userguide/publishing_maven.html#example_customizing_the_pom_file</p>
                   <p></p>
                 </li>
+                <details>
+                  <summary>How to fix</summary>
+                  <ul>
+
```

Comment:
Can we drop the superfluous blank lines?

Reply:
@sschuberth I had to undo the removal of the superfluous blank lines.
I think it's ok to keep them in the context of this PR as the output is identical with the how-to-fix sections for `rule violations` in that same file.

Reply:
> @sschuberth I had to undo the removal of the superfluous blank lines.
> I think it's ok to keep them in the context of this PR as the output is identical with the how-to-fix sections for `rule violations` in that same file.

Ok, but I'll file a follow-up issue to remove those lines from both places.